### DISCUSSION TITLE
2023/08/22/qixi-diary/

### DISCUSSION THREAD

**Discussion by giscus** [cyclinbox/cyclinbox.github.io#3] [General]
# 2023/08/22/qixi-diary/

七夕。 下午下起了雨，从哗啦哗啦到淅淅沥沥，这会儿雨停了，外面依然有点阴，看不到月亮和星光。

https://wz.anoms.top/2023/08/22/qixi-diary/

**Comment by cyclinbox** [comment]
夜晚容易多想。还是睡觉罢，梦里什么都有。
